src/components/motion/MotionController.cpp

```diff
@@ -53,8 +54,7 @@ bool MotionController::ShouldRaiseWake(bool isSleeping) {
 
 bool MotionController::ShouldShakeWake(uint16_t thresh) {
   /* Currently Polling at 10hz, If this ever goes faster scalar and EMA might need adjusting */
-  int32_t speed = std::abs(z + (y / 2) + (x / 4) - lastY / 2 - lastZ) / (time - lastTime) * 100;
-  //(.2 * speed) + ((1 - .2) * accumulatedSpeed);
```

Comment:
And I don't think you intended to remove this comment.

Reply:
Good catch, I must have accidentally deleted it when debugging.